package/packer/proxmox/cloud/.gitlab-ci.yml

```diff
@@ -0,0 +1,18 @@
+stages:
```

Comment:
Please remove this file `.gitlab-ci.yml`. It is fine to carry it as a patch in a fork but we don't use gitlab-ci and do not plan to.